scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -169,9 +169,9 @@ internal fun mapSnippetFindings(
                 val rawMatchedLinesSnippetFile = rawMatchedLines?.mirrorFile.orEmpty().collapseToRanges()
 
                 sourceLocation = rawMatchedLinesSourceFile.firstOrNull()
-                    ?.let { (startLine, endLine) -> TextLocation(it.file, startLine, endLine) }
-                snippetLocation = rawMatchedLinesSnippetFile.firstOrNull()
                     ?.let { (startLine, endLine) -> TextLocation(file, startLine, endLine) }
+                snippetLocation = rawMatchedLinesSnippetFile.firstOrNull()
```

Comment:
How about giving `it` the explicit `snippet` name in line 131 for clarity in a preceding commit? That would make this fix then more obvious.